__tests__/app.test.js

```diff
@@ -0,0 +1,45 @@
+const request = require('supertest');
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/");
+const connection = require("../db/connection");
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => connection.end());
+
```

Comment:
Both your beforeEach and AfterAll look sound - I would just wonder why you have chosen to use implicit return syntax in only one of them .Very minor but would make it consistent